doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,241 +63,400 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
+| DEVICE_NAME | String | Yes |   |
+| VENDOR | String | Yes |  |
+| DEVICE_MODEL | String | Yes |  |
+| DEVICE_SN | String | Yes |  |
+| BMC_IP | String | Yes |  |
+| SSH_USER | String | Yes |  |
+| SSH_PASSWORD | String | Yes |  |
+| ENABLE_PASSWORD | String | Yes |  |
+| PORT | List | Yes |  group multiple ports with same VLAN configuration, and separate different port group with ";" |
+| VLAN_TYPE | List | Yes | tag or untag |
+| VLAN_ID | List | Yes |  group multiple VLAN with same configuration requirements, and separate different VLAN group with ";" |
+| PORT_TYPE | List | Yes | trunk or access or hybrid  |
+
+<p align="center"><b>Table 6-3-8:</b> TOR VLAN information.</p>
 
-<p align="center"><b>Table 6-3-8-1:</b> Cluster information.</p>
 
+<a name="6.3.9"></a>
+### 6.3.9 EOR VLAN configuration information
+Multiple instances are expected, one instance for each EOR. 
 
 | Field # | type | mandatory | Instruction |
 |----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| device_name | String | Yes | indicate the device name|
-| storage_type | String | Yes | Ceph/HDFS/Swift/GFS/Luster, etc |
-| storage_os_user | String | No | storage os user|
-| storage_os_password | String | No | storage os password|
-| storage_network_address | String | Yes | storage access address|
-| storage_network_mask | String | Yes | mask for the network|
-| az_name | String | Yes | availabile zone which belongs to |
-| additional_attributes | List | Yes | specific attribute(key value pairs) list, decided by the storage_type. e.g: For example, Ceph would specific storage pool name, while HDFS would need replication options |
+| DEVICE_NAME | String | Yes |   |
+| VENDOR | String | Yes |  |
+| DEVICE_MODEL | String | Yes |  |
+| DEVICE_SN | String | Yes |  |
+| BMC_IP | String | Yes |  |
+| SSH_USER | String | Yes |  |
+| SSH_PASSWORD | String | Yes |  |
+| ENABLE_PASSWORD | String | Yes |  |
+| PORT | List | Yes |   group a list of ports with same VLAN configuration, and separate different port group with ";" |
+| VLAN_ID | List | Yes |  group multiple VLAN with same configuration requirements, and separate different VLAN group with ";" |
+| VLANIF_ADDRESS | List | Yes |  Vlanif addresses that need to be configured on EOR |
+| NETWORK_MASK | List | Yes |   |
+
+<p align="center"><b>Table 6-3-9:</b> EOR VLAN information.</p>
 
-<p align="center"><b>Table 6-3-8-2:</b> Storage device information.</p>
 
+<a name="6.3.10"></a>
+### 6.3.10 Host Aggregate
+Servers in the resource pool are divided to multiple HAs according to the difference from service or different hardware requirements.  
```

Comment:
"HAs"?

Reply:
HA = Host Aggregation group
I have updated the description, could you help to check if it's OK？

Reply:
HA = Host Aggregation group
I have updated the description, could you help to check if it's OK？